modules/15_climate/magicc7_ar6/postsolve.gms

```diff
@@ -17,9 +17,11 @@
 *' @code
 
 * Run only on first iteration to avoid incomplete GDXs
-if (iteration.val ge 2,
-* Run the climate assessment script. Takes around 2-3m for a single parameter set, including harmonization and infilling
 Execute_unload 'fulldata_postsolve';
+* if (iteration.val ge 2,
```

Comment:
I don't understand what is going on here. Why do you add an execute_unload statement in a comment. How can `iteration.val ge 1` ever be not satisfied?

Reply:
Ops, that was an oversight, that if can be removed now, a relic of when this was run only after the second iteration. And the `execute_unload` is not really in a comment, it writes a `fulldata_postsolve.gdx`, which has all the information at this point in the code. It would probably be wiser to write it in `main.gms` instead of inside a module, but the key here is that the actual `fulldata.gdx` is written too late for it to be used for the climate module.

Reply:
I mean [this comment](https://github.com/remindmodel/remind/pull/1713/files/7209af30f241c49a95df9f6319940a176f240cc4#diff-4de995e332ce8c89193de119283a705e349f3f303c42d3b99af2097aa134c845R24):
```
* Execute_unload 'fulldata_postsolve';
```

Reply:
Sorry, missed that too. The cluster is unbearably slow today, but a push removing it should appear at some point this afternoon